synthtool/transforms.py

```diff
@@ -203,7 +211,13 @@ def move(
         elif source not in excludes:
             # copy individual file
             if merge is not None and canonical_destination.is_file():
-                _merge_file(source, canonical_destination, merge)
+                try:
+                    _merge_file(source, canonical_destination, merge)
+                except Exception as e:
+                    logger.info(
+                        "_merge_file failed for %s: %s, fall back to copy", source, e
+                    )
+                    shutil.copy2(source, canonical_destination)
```

Comment:
I'm surprised we want to fall back to copy rather than fail completely. Have you seen a case where it's better to fall back to copy?